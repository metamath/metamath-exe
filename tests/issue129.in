open tex issue129.tex
show proof th1 /tex
close tex
tools
match issue129.tex "% This LaTeX file was created by Metamath on " N
exit
"cat issue129.tex"
