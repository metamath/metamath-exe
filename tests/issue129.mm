$c A $.

ax-1 $a A A A A A $.

th1 $p A A A A A $= ax-1 $.

$( $t
    latexdef "A" as "\text{ A A A A A A A A A A }";
$)
