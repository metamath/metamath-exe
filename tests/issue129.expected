MM> READ "issue129.mm"
Reading source file "issue129.mm"... 121 bytes
121 bytes were read into the source buffer.
The source has 3 statements; 1 are $a and 1 are $p.
No errors were found.  However, proofs were not checked.  Type VERIFY PROOF *
if you want to check them.
MM> Continuous scrolling is now in effect.
MM> Created LaTeX output file "issue129.tex".
Reading definitions from $t statement of issue129.mm...
1 typesetting statements were read from "issue129.mm".
MM> Outputting proof of "th1"...
The LaTeX source was written to "issue129.tex".
MM> The LaTeX output file "issue129.tex" has been closed.
MM> Entering the Text Tools utilities.  Type HELP for help, EXIT to exit.
TOOLS> The input had 28 lines, the output has 27 lines. First output line:
\documentclass{article}
TOOLS> Exiting the Text Tools.  Type EXIT again to exit Metamath.
MM> \documentclass{article}
\usepackage{amssymb} % amssymb must be loaded before phonetic
\usepackage{phonetic} % for \riota
\usepackage{mathrsfs} % for \mathscr
\usepackage{mathtools} % loads package amsmath
\usepackage{amsthm} % amsthm must be loaded after amsmath
\usepackage{accents} % accents should be loaded after mathtools
\theoremstyle{plain}
\newtheorem{theorem}{Theorem}[section]
\newtheorem{definition}[theorem]{Definition}
\newtheorem{lemma}[theorem]{Lemma}
\newtheorem{axiom}{Axiom}
\allowdisplaybreaks[1] % Allow page breaks in {align}
\usepackage[plainpages=false,pdfpagelabels]{hyperref}
\hypersetup{colorlinks} % Get rid of boxes around links
\begin{document}

\begin{proof}
\begin{align}
  1 &&  & \text{ A A A A A A A A A A } \text{ A A A A A A A A A A }
    \notag \\ && & \qquad \text{ A A A A A A A A A A }
    \notag \\ && & \qquad \text{ A A A A A A A A A A }
\text{ A A A A A A A A A A }&(\mbox{\ref{eq:ax-1}})\notag
\end{align}
\end{proof}

\end{document}
